Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = "1.0.203"
 thiserror = "1.0.61"
 tracing = "0.1.40"
 url = "2.5.2"
+tokio = { version = "1", features = ["time"] }
```

Comment:
I guess if `PendingTransactionBuilder` will work, we will be able to remove this dependency.

Reply:
Yes, removed the dependency since now it's not needed anymore.